vistautils/range.py

```
@@ -687,11 +687,11 @@ def range_containing(self, value: T) -> Optional[Range[T]]:
         raise NotImplementedError()
 
     @abstractmethod
-    def range_enclosing_range(self, value: Range[T]) -> Optional[Range[T]]:
+    def range_enclosing_range(self, rng: Range[T]) -> Optional[Range[T]]:
```

Was there a compelling test failure that required changing the parameter names? Otherwise given this is a published packaged I'd refrain from doing so incase anyone is passing in parameters by keyword.

---

There was a pylint error complaining that the child classes (e.g. `_SortedDictRangeSet`) had function parameters which overwrote the abstract method's names (in this case,` _SortedDictRangeSet.range_enclosing_range()` used `rng` as a parameter where the abstract class used `value`. In hindsight, I don't think this should require a change at all since it didn't cause an error in the Python 3.6 environment I set up for testing. I think this is something which was ignored prior but is now raised as an issue since I updated the pylint version to work with Python 3.8

---

Ah! Well ok, this is probably something that should change but doing so is breaking compatibility so let's just ignore the error for now.

---

Ok, I can revert the classes to how they were before and ignore the errors.

---

The parameters are now back to how they were before and the corresponding child class functions are set up to ignore the specific error pylint was raising (`# pylint: disable=W0237`)